src/index.ts

```diff
@@ -181,8 +207,11 @@ async function isPodmanLocalImageLatest(): Promise<boolean> {
 // remove the pulled image from the Podman image storage
 async function removeDockerImage(): Promise<void> {
     if (imageToPush) {
-        core.info(`Removing ${imageToPush} from the Podman image storage`);
-        await execute(await getPodmanPath(), [ "rmi", imageToPush ]);
+        for (const tag of tagsList) {
+            const imageWithTag = `${imageToPush}:${tag}`;
+            await execute(await getPodmanPath(), [ "rmi", imageWithTag ]);
+            core.info(`Removing ${imageWithTag} from the Podman image storage`);
```

Comment:
Can you put this msg before and outside the loop ?

`Removing ${imageToPush} from the Podman image storage`

then you can see the podman output as it untags/removes each image.